src/utils.py

```diff
@@ -204,11 +205,20 @@ def download_chain_spec(url: str, filename: Path) -> None:
         c.CHAIN_SPEC_DIR.mkdir(parents=True)
     try:
         download_file(url, Path(c.CHAIN_SPEC_DIR, filename))
+        validate_file(Path(c.CHAIN_SPEC_DIR, filename), file_type='json')
     except ValueError as e:
         logger.error(f'Failed to download chain spec: {e}')
-        raise e
+        raise ValueError(e)
 
 
+def validate_file(filename: Path, file_type: str):
+    if file_type == 'json':
+        try:
+            file_obj = open(filename, 'r')
+            _ = json.load(file_obj)
+        except json.JSONDecodeError as e:
+            raise ValueError(f"Validating chain spec {filename} failed with error: {e}")
```

Comment:
If the goal here is to build an "exception path" specifically for spec file issues, why not create a new error type `SpecFileError` instead of using the generic `ValueError`?

Reply:
> If the goal here is to build an "exception path" specifically for spec file issues, why not create a new error type `SpecFileError` instead of using the generic `ValueError`?

I tried this, but couldn't get the error message to show in the status. Didn't figure out what I was doing wrong, but I could push a branch with my implementation and you could help me take a look, if you can. I believe a `ValueError` will suffice, for now at least